### PR TITLE
Modify typo

### PR DESCRIPTION
**Pull request by roth1002**
rackt -> reactjs

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -53,18 +53,18 @@ ReactDOM.render(
 
 Connects a React component to a Redux store.
 
-It does not modify the component class passed to it.
```
Comment (gaearon) [inline docs/api.md]: Spaces are significant in markdown.
Two spaces before the line break make newlines act like newlines.

Can you please remove all spacing changes?

Comment (roth1002) [inline docs/api.md]: OK,  I will fixed it :)

File: docs/api.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ Returns the wrapped component instance. Only available if you pass `{ withRef: t
 
 * It does not modify the passed React component. It returns a new, connected component, that you should use instead.
 
-* The `mapStateToProps` function takes a single argument of the entire Redux store’s state and returns an object to be passed as props. It is often called a **selector**. Use [reselect](https://github.com/rackt/reselect) to efficiently compose selectors and [compute derived data](http://rackt.github.io/redux/docs/recipes/ComputingDerivedData.html).
+* The `mapStateToProps` function takes a single argument of the entire Redux store’s state and returns an object to be passed as props. It is often called a **selector**. Use [reselect](https://github.com/reactjs/reselect) to efficiently compose selectors and [compute derived data](http://redux.js.org//docs/recipes/ComputingDerivedData.html).
```
Comment (gaearon) [inline docs/api.md]: There is a double slash here (`(http://redux.js.org//docs`)

**Comment by gaearon** [comment]
Would you please fix the merge conflict and the double slash? Thanks.

**Comment by gaearon** [comment]
Closing as inactive but I’m happy to reopen if the above comments are addressed.

**Pull request closed without merging**